src/devices/src/virtio/vsock/persist.rs

```diff
@@ -120,6 +120,11 @@ where
         } else {
             DeviceState::Inactive
         };
+
+        if state.virtio_state.activated {
+            vsock.send_transport_reset_event().unwrap();
```

Comment:
We should handle/propagate the possible errors here, instead of just using `unwrap()`.